sunpy/map/mapsequence.py

```diff
@@ -271,6 +271,12 @@ def plot(self, axes=None, resample=None, annotate=True,
         plot_function : function
             A function to be called as each map is plotted.
             For more information see `sunpy.visualization.animator.MapSequenceAnimator`.
+        norm : string 
+            Normalization used in scaling the plot.
+            This will override the predefined value in ``plot_settings['norm']``.
+        cmap : string 
```

Comment:
It will be https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.set_cmap.html